docs/architecture/input-parameters-on-runtime.md

```diff
@@ -0,0 +1,23 @@
+# Input parameters for API, Event Definitions or API Packages in the Runtime
+
+## Overview
+
+Previously, in Kyma Runtime, while provisioning Service Class created from API/Event Definition, there were no possibility to pass input parameters. The ability may be useful in some specific use cases. This document describes passing input parameters from Kyma Runtime to Application or Integration System.
+
+## Solution
+
+The solution utilizes labels on Application level. User provides JSON schema for Service Class `instanceCreateParameterSchema` parameter by creating new Label Definition in the following format:
+
+ **Key**: 
+- For API Definition: `instance-create-params.apis.{api-def-name}`
+- For Event Definition: `instance-create-params.{event-def-name}.instance-create-params`
+- For API Package: `instance-create-params.api-packages.{api-package-name}`
+    
+**Value**: JSON schema for the `instanceCreateParameterSchema` for given Service Class
+
+On Kyma Runtime, Runtime Agent queries Applications for given Runtime. It reads all labels with prefix `instance-create-params` and updates `instanceCreateParameterSchema` properties in Service Class list to create.
+
+## Client Usage
+
+To be defined later.
```

Comment:
Define clearly that we still need to define how we will store, many instances of input parameters in compass/application itself.